### ISSUE TITLE
Dynamically add content for the graphics section

### ISSUE THREAD

**Issue by jdwilkin4** [jdwilkin4/contrast-ratio-repo#46]
This is what we currently have for the graphics section
<img width="126" alt="Screen Shot 2022-08-09 at 5 57 11 AM" src="https://user-images.githubusercontent.com/67210629/183652536-43e15143-8b98-4fc7-8a0b-dd8728073412.png">

Using the existing hex code functionality, we should display pass if the contrast ratio is good and fail if the contrast ratio is poor. 
The background colors of green and red should change based on the condition. 

We want to match the functionality for the WebAIM application
https://webaim.org/resources/contrastchecker/
<img width="232" alt="Screen Shot 2022-08-09 at 6 04 43 AM" src="https://user-images.githubusercontent.com/67210629/183654051-6b94c524-93fd-4126-9877-8cd6b65460f1.png">
